docs/_scss/components/_accordions.scss

```diff
@@ -11,21 +11,6 @@ $accordion-border: 3px solid $color-gray-lightest;
     margin-top: 1rem;
```

Comment:
@joshbruce first off, did that solution work for you at all?

Based on the documentation on npm, and what I've read, there doesn't seem to be a large difference(http://stackoverflow.com/questions/12478679/npm-install-vs-update-whats-the-difference), I gave you that command just to ensure we're both doing the same thing to make debugging a little easier.